composeApp/build.gradle.kts

```diff
@@ -125,9 +129,10 @@ sqldelight {
             srcDirs.setFrom("src/androidMain/sqldelight/app")
         }
         // Internal DB containing TMDB cache
-        create("Tmdb") {
+        create("TmdbCache") {
             packageName.set("io.github.couchtracker.db.tmdb")
             srcDirs.setFrom("src/androidMain/sqldelight/tmdb")
```

Comment:
Question: should we change the name of the package as well since we changed the name of the DB?